src/utils/drops.ts

```diff
@@ -0,0 +1,117 @@
+import BigNumber from "bignumber.js";
```

Comment:
In xrpl-py, this file is called `xrp_conversions`. That sounds like a better file name than `drops`.